### ISSUE TITLE
TA ssh key is only readable by root

### ISSUE THREAD

**Issue by romain-jacob** [nsg-ethz/mini_internet_project#40]
Currently, when the mini-internet is built, the ssh key located in `/groups` is only readable by root, hence is only usable to ssh into the student containers using `sudo`.

I find it a bit odd and easy to fix. But I wonder: is there a (security?other?) reason behind it?

**Comment by NotSpecial** [comment]
I don't believe there is any particular reason for it, we usually never use the key on the server itself. On the server itself, most actions are easier to perform using `docker exec`.

So usually we share this key via chat with the assistants, so the permission issue doesn't really come up.
